### DISCUSSION TITLE
В последней версией конфликт с hd models pack ?

### DISCUSSION THREAD

**Discussion by Darkkys** [OGSR/OGSR-Engine#395] [Q&A]
На последней версии огср в лог спамит ошибка, как понимаю от хд-моделей нпц.
В версии огср 2.1.79 такого не было.

В чем дело и как исправить?
![S T A L K E R   Call of Pripyat Screenshot 2022 01 09 - 14 34 36 20](https://user-images.githubusercontent.com/97330663/148673542-182c77a2-a8d3-4feb-88c4-a9c4e3fd6b91.png)

**Comment by xrSimpAI** [comment]
Это уже год как.
Это сообщение говорит о том, то в этих omf инверсная кинематика включена, но настроена не правильно.

**Reply by xrSimpAI** [reply]
и что это за hd models pack ?

**Reply by Darkkys** [reply]
Это критично? Можно ли исправить, или хотя бы отрубить спам в лог?

**Reply by Darkkys** [reply]
насколько помню - вот этот https://www.moddb.com/mods/shadow-of-chernobyl-hd-models-addon

**Reply by xrSimpAI** [reply]
я бы рекомендовал избавиться от fat_npc_animation - и прописать обычный stalker_animation и желательно с поддержкой кинематики. (ссылки на них тоже в wiki должны быть)
Например так если в модели не хочется лезть:
![image](https://user-images.githubusercontent.com/16670637/148674286-81ce22b1-d10d-4688-8216-25d98f3abeed.png)

**Comment by Darkkys** [comment]
А нельзя просто отключить логирование этой ошибки?

**Reply by bukhaan** [reply]
Чувак, ты смог кинематику настроить?

**Comment by xrSimpAI** [comment]
Если отключить, можно не узнать, что у тебя не работает кинематика, и собственно почему не работает.

**Reply by Darkkys** [reply]
Просто я сам этого не сделаю, автор хд моделса скорее всего не станет заморачиваться.. А так за этим спамом другие ошибки можно пропустить )

**Reply by xrSimpAI** [reply]
А что сложного? в вики есть способ как анимации для модели подменить. обыкновенный stalker_animation пропиши и всё
